### PR TITLE
fixed packet_to_search_body for unicode strings

### PR DESCRIPTION
**Pull request by Beisenbek**
Problem with saving unicode message to riakkv mam with exception badarg in list_to_binary function

### CODE REVIEW

File: apps/ejabberd/src/mod_mam_riak_timed_arch_yz.erl [hunk 1]
```diff
@@ -241,7 +241,9 @@ create_obj(Host, MsgId, SourceJID, Packet, Type) ->
             pm -> mod_mam;
             muc -> mod_mam_muc
         end,
-    BodyValue = list_to_binary(mod_mam_utils:packet_to_search_body(ModMAM, Host, Packet)),
+
```
Comment (gadgetci) [inline apps/ejabberd/src/mod_mam_riak_timed_arch_yz.erl]: According to **Elvis**:
> Line 244 has 9 trailing whitespace characters.

**Comment by michalwski** [comment]
I reopened to trigger new travis build on top of current master branch.

**Comment by Beisenbek** [comment]
@michalslaski something wrong with the travis build.

**Comment by michalwski** [comment]
It was a random failure. Now it's green.

**Comment by Beisenbek** [comment]
@michalwski how long it will take to merge with the master?

**Comment by michalwski** [comment]
I just merged it, thanks!

**Pull request merged**